go/vumitools/api.py

```diff
@@ -181,9 +179,10 @@ def running_conversations(self):
 
     @Manager.calls_manager
     def draft_conversations(self):
-        # TODO: Get rid of this once the old UI finally goes away.
+        # TODO: This should probably be `stopped_conversations` instead, but we
+        #       still apparently use `draft` in the UI in places.
         conversations = yield self.active_conversations()
-        returnValue([c for c in conversations if c.is_draft()])
+        returnValue([c for c in conversations if c.stopped()])
```

Comment:
`is_draft` doesn't have much meaning anymore and it's functionally equivalent to `stopped` if archived conversations are excluded, which is the case here.